docs/source/python.rst

```diff
@@ -80,6 +80,55 @@ This image can be viewed in `napari` using the
     $ napari test_ngff_image.zarr
 
 
+Writing HCS OME-NGFF images
+===========================
+
+This sample code shows how to write a high-content screening dataset (i.e. culture plate with multiple wells) to a OME-NGFF file::
+
+    import numpy as np
+    import zarr
+
+    from ome_zarr.io import parse_url
+    from ome_zarr.writer import write_image, write_plate_metadata, write_well_metadata
+
+    path = "test_ngff_image.zarr"
```

Comment:
This should be `file_path` and `plate` makes more sense than `image` here.
e.g. `file_path = "test_ngff_plate.zarr"`

Reply:
Thanks for the catch! I missed this when converting my code to match the style of the documentation. I instead changed `file_path` to `path` to match the other sections. Now fixed.